tools/run-dev.py

```diff
@@ -8,6 +8,8 @@
 from typing import Any, Callable, Generator, List, Sequence
 from urllib.parse import urlunparse
 
+from django.conf import settings
```

Comment:
Hmm, do we intentionally avoid importing Django here?  I think doing so may materially slow the startup process for the development server.

Why isn't the fix to just remove `proxy_port` here?

Reply:
> Hmm, do we intentionally avoid importing Django here? I think doing so may materially slow the startup process for the development server.

Ok. I will avoid importing `django` here.

> Why isn't the fix to just remove `proxy_port` here?

I think so specifying `proxy_port` will make it easier for new users to connect and especially those who are using remote setup.